fuzzers/fuzz_parser.c

```diff
@@ -0,0 +1,36 @@
+#include <stdint.h>
+#include <stdlib.h>
+#include <string.h>
+#include "http_parser.h"
+
+int LLVMFuzzerTestOneInput(const uint8_t *data, size_t size){
+    // prepare a null-terminated string.
+    char *new_str = (char *)malloc(size+1);
```

Comment:
Does it have to be a null-terminated string? I don't think the parser cares as long as you provide right `size`.

If it doesn't - is there any point in copying the input?

Reply:
You are right! Fixed.